lib/ProductOpener/Food.pm

```diff
@@ -367,6 +366,39 @@ unit_to_g(520,mg) => returns 0.52
 
 =cut
 
+# This is a key:value pairs
+# The keys are the unit names and the values are the multipliers we can use to convert to a standard unit. 
+# We can divide by these values to do the reverse ie, Convert from standard to non standard
+my %unit_conversion_map = (
```

Comment:
I like this approach. I think it could be improved by separating the translation data from the conversion data.

e.g. we could have two maps:

%unit_translation_map = (
"gram" => "g",
"grams" => "g",
"gramme" => "g",
"grammes" => "g",
);

And then a conversion map:

%unit_conversion_map = (
"g" => 1,
"kg" => 1000,
);

But we can do it in another PR.

Reply:
I agree this approach is nice, will do it in another PR